tools/vast-regenerate/main.cpp

```diff
@@ -93,10 +93,8 @@ caf::error write_index_bin(const std::vector<vast::uuid>& uuids,
   for (const auto& uuid : uuids) {
     if (auto uuid_fb = pack(builder, uuid))
       partition_offsets.push_back(*uuid_fb);
-    else {
-      fmt::print(stderr, "failed to pack uuid {}: {}\n", uuid, uuid_fb.error());
-      return 1;
-    }
+    else
+      return uuid_fb.error();
```

Comment:
We are losing the context here("failed to pack uuid"), but the proper error message is returned to the caller; this is ok as it is or maybe:

```
      return caf::make_error(vast::ec::convert_error,
                             fmt::format("failed to pack uuid {}", uuid),
                             uuid_fb.error());
```

Reply:
All `pack` overloads share `caf::error` as return type. The one for `uuid` can't fail though, and I don't see how that would change in the future.